_config.yml

```
@@ -3,6 +3,7 @@ page_title                                :
 
 # App Info
 ios_app_id                                : 1234793120                                # Required. Enter iOS app ID to automatically populate name, price and icons (e.g. 718043190).
+ios_app_country                           : us                                        # Optional, Enter 2 letter country code as in https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2
```

I'd change `Optional` to `Required`, as it is required for apps in non-US AppStores.

```suggestion
ios_app_country                           : us                                        # Required outside USA. Enter 2 letter country code as in https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2
```